Sources/SwiftNavigation/NSObject+Observe.swift

```diff
@@ -121,13 +144,34 @@
     public func observe(
```

Comment:
Probably an important change, now since we provide isolation in `_observe`, I removed `DispatchQueue.async`